packages/form-js-editor/src/features/properties-panel/entries/factories/simpleBoolEntryFactory.js

```diff
@@ -1,5 +1,5 @@
 import { get } from 'min-dash';
-import { CheckboxEntry, isCheckboxEntryEdited } from '@bpmn-io/properties-panel';
+import { ToggleSwitchEntry, isToggleSwitchEntryEdited } from '@bpmn-io/properties-panel';
```

Comment:
That leads existing entries to be toggle switches, as `searchable`

![image](https://github.com/bpmn-io/form-js/assets/9433996/fefa5107-0d4d-4437-bc61-682a11d9936a)

I guess it's intended?